normandy/recipes/management/commands/update_addon_urls.py

```diff
@@ -0,0 +1,52 @@
+import json
+
+from urllib.parse import urlparse, urlunparse
+from django.core.management.base import BaseCommand
+
+from normandy.recipes.models import RecipeRevision
+
+
+class Command(BaseCommand):
+    """
+    Rewrite all revisions to update the URL of add-ons stored in Normandy's
+    file storage to a new hostname.
+    """
+
+    help = "Updates add-on URL in revisions"
+
+    def add_arguments(self, parser):
+        parser.add_argument("new_hostname", help="Hostname to convert add-on urls to")
+
+    def handle(self, *args, new_hostname, **options):
```

Comment:
I've always done ...

```python
def handle(self, **options):
    new_hostname = options["new_hostname"]
```

But come to think of it I can't see what the difference is. 

[The docs](https://docs.djangoproject.com/en/2.1/howto/custom-management-commands/) use `options`.